src/htmlhighlighter.test.js

```diff
@@ -625,4 +647,27 @@ describe('HTML Highlighter', function() {
       attest.totalHighlights(1, 1);
     });
   });
-});
+}
+
+function describeFullDocumentTests() {
+  if (BROWSER) {
+    throw new Error('Full document tests not available in browser mode');
```

Comment:
Same comment here about when we expect this to be false.

Reply:
This is the only exception produced in this test spec function. 

Essentially, `BROWSER` is `true` when `yarn run test` is executed -- see difference between `webpack.config.js` and `webpack.config-test.js`.

Reply:
@miguel-guedes does that imply that `yarn run test` uses `webpack.config.js` and _not_ `webpack-test.config.js`? It's `webpack.config.js` that has `BROWSER` set to `true` as far as I can tell.

Reply:
@kujenga bah, I meant it is `false` in the condition above.

Reply:
`true` when `webpack.config.js` is used.

Reply:
Ah, okay! sounds like we're on the same page then, thanks 👍

Reply:
IOW, full document tests can only ever run in jsdom environment.